examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<blockquote>use register and perform activities with <code>PASEOS</code></blockquote><p>-&gt; use PASEOS to perform activities.</p><p> </p><blockquote>fetection</blockquote><p>the detection</p><p> </p><blockquote>to this aim</blockquote><p>Twice in a row "To this aim" , second sentence also missing a verb</p><p> </p><blockquote>leverages on bands</blockquote><p>leverages the spectral bands B8A , ... </p><p> </p><blockquote> and perform pixel-level detection</blockquote><p>to perform</p><p> </p><blockquote>regardless their </blockquote><p>regardless of their</p><p> </p><blockquote>geografical</blockquote><p>geographical</p><p> </p><blockquote>We thanks</blockquote><p>We thank</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>